pkg/endpoint/endpoint.go

```diff
@@ -352,6 +352,10 @@ func NewEndpointFromChangeModel(base *models.EndpointChangeRequest, l pkgLabels.
 		return nil, nil
 	}
 
+	if lbls := l.FindReserved(); lbls != nil {
+		return nil, fmt.Errorf("Cannot create endpoint with labels %s", lbls)
```

Comment:
Please make this error more descriptive. `Not allowed to create endpoint with reserved labels %s` or similar

Reply:
Done